docs/speech-to-text/features/diarization.mdx

```diff
@@ -252,3 +269,84 @@ If you do not specify any labels then defaults will be used (e.g. Channel 1). Th
   }
 ]
 ```
+
+### Realtime usage
+
+Realtime channel diarization is useful for remote conference calls, where separate audio streams may include multiple speakers.  
+
+The maximum number of channels depends on whether you use a multi-session container and its connection limits.  
+
+To enable the feature, set the `diarization` property to `channel` or `channel_and_speaker`, and define each channel name under `channel_diarization_labels`.
+
+```json
+{
+  "type": "transcription",
+  "transcription_config": {
+    "language": "en",
+    // highlight-start
+    "diarization": "channel_and_speaker",
+    "channel_diarization_labels": ["New_York", "Shanghai", "Paris"]
+    // highlight-end
+  }
+}
+```
+
+The Transcriber confirms by including the channel diarization labels in the `channels` field of the `RecognitionStarted` message:
```

Comment:
I think this is too technical. It might be better phrased in a more user-friendly way, e.g.:

```suggestion
When you use channel diarization, you should see an additional `channels` field in the `RecognitionStarted` message which lists all the channels you requested:
```